### PR TITLE
Add Async fs and sync fs and add txt file to read and write

### PR DESCRIPTION
**Pull request by rishavquventix**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -13,5 +15,38 @@ app.use(express.json());
...
 
-app.listen(config.port, () => console.log(`Server Up and running`));
+//Asynchronous Read, Write and append file    
+fs.readFile("./fs.txt", "utf8", (err, data) => {
+    if (err) {
+        console.log(err);
+    } else {
+        console.log(data);
+    }
+});
```
Comment (officialanurag) [inline app.js:25]: You are supposed to do it with async/await.
**Do it with async/await?**

File: app.js [hunk 2]
```diff
@@ -13,5 +15,38 @@ app.use(express.json());
...
+
+fs.appendFile("./fs.txt", "Hiiiii", (err) => {
+    if (err) {
+        console.log(err);
+    } else {
+        console.log('Writing data');
+    }
+});
+
```
Comment (officialanurag) [inline app.js:42]: Use async/await instead of callbacks.

File: app.js (hunk 1 above)
Comment (rishavquventix) [reply app.js:25]: Done Sir

File: app.js (hunk 2 above)
Comment (rishavquventix) [reply app.js:42]: Done Sir

**Pull request closed without merging**